src/format/format.ts

```diff
@@ -14,6 +14,10 @@ import { NormalizedProblem, ProblemSeverity, LineColLocationObject, LocationObje
 import { getCodeframe, getLineColLocation } from './codeframes';
 import { Totals } from '../cli';
 
+const ERROR = {
```

Comment:
I would rename to:

```suggestion
const ERROR_MESSAGES = {
```

But is there a point to have it separately? It is not reused anywhere. I'm okay to keep it though.